adaptive_experimental/lib/navigation_rail.dart

```diff
@@ -105,9 +105,9 @@ class _NavigationRailState extends State<NavigationRail> with TickerProviderStat
   Widget build(BuildContext context) {
     final leading = widget.leading;
     return DefaultTextStyle(
-      style: TextStyle(color: Theme.of(context).colorScheme.primary),
+      style: TextStyle(color: Theme.of(context)!.colorScheme.primary),
```

Comment:
nit: could you factor out `colorScheme`?

Reply:
Good idea!

Reply:
Done.